background.ts

```diff
@@ -25,9 +27,12 @@ import buildApplicationMenu from '@/main/mainmenu';
 
 Electron.app.setName('Rancher Desktop');
 
+// Maybe let people switch this on the fly
+const ImageProviderName = 'nerdctl';
```

Comment:
We'll need to update this when we also provide Docker. It'll be related to the engine they're using. We can update all of this when we provide Docker/Moby as an alternative.

Reply:
That's the idea. Write a backend, make necessary changes to the UI, and change this field.